src/util/key_load.c

```diff
@@ -238,11 +239,18 @@ int key_load(charbuf key_id)
         break;
       }
 
-      pelz_log(LOG_DEBUG, "Common Name: %.*s, %d", common_name.len, common_name.chars, common_name.len);
-      pelz_log(LOG_DEBUG, "Port Number: %d", port);
+      //TODO: Change this to be an input from somewhere
+      client_name = new_charbuf(10);
```

Comment:
What is ```client_name``` used for? Is it something that will come from the application that's using ```pelz```?

Reply:
The ```enclave_retrieve_key``` function needs pelz's public certificate for the ECDH handshake with the proxy key server.  I hard coded the ```client_name``` based on how we create pelz's public certificate. I will create an issue to do this change to the pelz code.